### ISSUE TITLE
Add option to enable/disable CSD

### ISSUE THREAD

**Issue by Mau-G** [midori-browser/core#216]
CSD is mostly a GNOME feature and some users feel that don't blend in well in other GTK+ environments such as Xfce, Mate and Cinnamon, which do not use CSD by default.

An option to toggle CSD would allow users to choose what they prefer. Programs such as Tilix have such option. See Screenshot
![screenshot_2018-12-20_23-39-47](https://user-images.githubusercontent.com/44322817/50294589-310a6c80-04b1-11e9-86ba-536d4c92e643.png)

**Comment by asc-soc** [comment]
Midori honours the GTK_CSD environment variable, see #4 .

**Comment by Mau-G** [comment]
> Midori honours the GTK_CSD environment variable, see #4 .

I am aware but an option in the setting to toggle it just for Midori would be more intuitive for users. 
For example an option in the setting add these options for the desktop file:

`Exec=env GTK_CSD=0 midori %u`
`Exec=env GTK_CSD=1 midori %u`

**Comment by kalikiana** [comment]
I agree with @Mau-G actually. `GTK_CSD` works well as a default but users should be able to discover this in the UI. My only concern is making sure that switching works at runtime - I consider "restart required" a big no in good design (that should go into the HIG once I get to writing them, or somebody else beats me to it).

**Comment by kalikiana** [comment]
Perhaps we actually wanna follow the trend and [pick the default depending on the desktop environment](https://git.xfce.org/apps/catfish/commit?id=75dce159cb602d9d2c6f23d937a2e10e8080e613)?

That is, I still agree we should add an option for it, but I'm adding this here as semi-related food for thought and it'll be a pretty easy fix _if someone happens to wanna take a stab at their first Midori commit_ 😉

**Comment by Mau-G** [comment]
> Perhaps we actually wanna follow the trend and [pick the default depending on the desktop environment](https://git.xfce.org/apps/catfish/commit?id=75dce159cb602d9d2c6f23d937a2e10e8080e613)?
> 
> That is, I still agree we should add an option for it, but I'm adding this here as semi-related food for thought and it'll be a pretty easy fix _if someone happens to wanna take a stab at their first Midori commit_ wink

I was actually going to message you about Catfish and then forgot. Good that you spotted it. 
I think it is a good way to do it and easy enough to modify the config file to toggle the setting manually. I might look into it for a commit to Midori.

**Comment by Mau-G** [comment]
Hi Chris, 
Thanks for the merging commit #258. I think now there is even more reason to implement  issue  #204?
just my 2cts.

**Comment by kalikiana** [comment]
Not sure how they're related 🤔 Since we're in feature freeze I think we should probably leave for the next release but I'm on board with going that direction
